tests/js/spec/views/twoFactorRequired.spec.jsx

```diff
@@ -7,7 +7,7 @@ import AccountSecurityWrapper from 'app/views/settings/account/accountSecurity/a
 
 const ENDPOINT = '/users/me/authenticators/';
 const ORG_ENDPOINT = '/organizations/';
-const PENDING_INVITE = 'pending-invite';
+const INVITE_COOKIE = 'pending-invite';
```

Comment:
Why did this change here?

Reply:
The const name just makes a bit more sense here I think, since it indicates it's a cookie instead of just the cookies name

Reply:
Sure, just want it consistent. Can this change in invite_helper.py as well

Reply:
Yeah good catch